### DISCUSSION TITLE
How do you train CodeProject.AI to do facial recognition?

### DISCUSSION THREAD

**Discussion by doctorttt** [skrashevich/double-take#97] [General]
Hi, I just installed this fork of Doubletake because it supports CodeProject.AI. This is my first time using CodeProject.AI, and I'm using the latest gpu version. How do I train CodeProject.AI to recognize faces?  I came from Compreface, which has a very straightforward gui to upload faces, but I'm not sure how to do the same in CodeProject.AI. Apologies for asking this question here.

**Comment by jmtatsch** [comment]
Use double take itself to train faces. See train tab.

**Comment by doctorttt** [comment]
Thanks. I just tested the train tab in Double-take and it does pass the facial images to CodeProject.AI for training.

**Comment by sirhideo** [comment]
@doctorttt does CodeProject.AI process faster than Compreface? I use the core Arcface-r100 and it takes between 4-10 seconds to process a picture... I am an asian so facenet isn't quite accurate.
